FlightStreamDeck.SimConnectFSX/SimConnectFlightConnector.cs

```diff
@@ -241,6 +241,11 @@ public void AvMasterToggle(uint state)
             SendCommand(EVENTS.AVIONICS_TOGGLE, state);
         }
 
+        public void GenericSet(TOGGLE_EVENT evt, uint data)
```

Comment:
Can we use existing Trigger function for this? This looks pretty much identical.

Reply:
I swear the data = 0 was causing an issue when I was testing it earlier, but I just tried it using Trigger, and it works.  So its updated.